src/test/java/com/icthh/xm/tmf/ms/communication/config/SmppTest.java

```diff
@@ -32,7 +32,7 @@
 @SpringBootTest(classes = {CommunicationApp.class})
 public class SmppTest {
 
-    public static final String DOCKER_IMAGE_NAME = "smpp-emulator";
+    public static final String DOCKER_IMAGE_NAME = "jbsukraine/smpp-emulator";
```

Comment:
Why not use xm-online?

Reply:
reasonable comment!
@qlimenoque please move it to xm-online docker repo.